docs/basic_configuration.rst

```diff
@@ -0,0 +1,66 @@
+There are much more options, see [[configuration]].
+
+Preparations
+============
+Get a supported map package
```

Comment:
Don't you need a blank line between the titles to get a proper rendering?

Reply:
Not with sphinx's RST it seems : https://navit.readthedocs.io/en/latest/basic_configuration.html